amr-wind/equation_systems/vof/volume_fractions.H

```diff
@@ -442,9 +442,10 @@ AMREX_GPU_HOST_DEVICE AMREX_FORCE_INLINE bool interface_band(
     amrex::Real VOF_max = 0.0;
     amrex::Real VOF_min = 1.0;
     bool VOF_mid = false;
-    for (int ii = -1; ii < 2; ++ii) {
-        for (int jj = -1; jj < 2; ++jj) {
-            for (int kk = -1; kk < 2; ++kk) {
+    constexpr int n_band = 1; // must be 1, 2, or 3
```

Comment:
Would we ever need n_band other than 1 ?

Reply:
I'm not sure, but if we run into issues, trying different values of n_band allows us to trade some dissipation for increased robustness. So it could be helpful in the future, I don't know.

Reply:
Sounds good!